### PR TITLE
Add Findutils

### PR DESCRIPTION
**Pull request by pmiess**
Add gnuwin32 package of findutils.

This is a very broadly supported project with 175 packages in 116 repos.

This package is need by the ttf2pt1 gnuwin32 package

I have added an alternative implementation from ezwinports

This is part of 0install/0install.de-feeds#3

Moved from https://github.com/0install/repo.roscidus.com/pull/200

### CODE REVIEW

File: utils/findutils.xml [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+      <requires interface="https://apps.0install.net/utils/coreutils.xml">
+        <environment name="PATH" insert="bin" />
+      </requires>
+      <requires interface="https://apps.0install.net/utils/mktemp.xml">
+        <executable-in-path name="mktemp" />
+      </requires>
+      <requires interface="https://apps.0install.net/utils/bash.xml">
+        <executable-in-var name="SHELL" />
+      </requires>
+      <requires interface="/https://apps.0install.net/utils/sed.xml">
```
Comment (bastianeicher) [inline utils/findutils.xml:32]: There's a leading `/` in the URL here that needs to be removed.

File: utils/findutils.xml [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+       updated and  the  directories  for  which  they  contain
+       entries  depend  on  how often updatedb is run, and with
+       which arguments.
+</description>
+  <entry-point command="updatedb" binary-name="updatedbbat">
+    <needs-terminal />
+    <name xml:lang="en">updatedb bat</name>
+    <summary xml:lang="en">This is a pale imitation of the original `updatedb.sh'</summary>
+  </entry-point>
+  </entry-point>
```
Comment (bastianeicher) [inline utils/findutils.xml:131]: This `</entry-point>` needs to be moved up to line 126.

File: utils/findutils.xml [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+  <name>FindUtils</name>
+  <summary xml:lang="en">FindUtils: find and then operate on files</summary>
+  <description xml:lang="en">`find' is a program which searches a directory tree to find a file or group of files. It walks the directory tree and reports all occurences of a file matching the user's specifications. 
+
+`locate' scans one or more databases of filenames and displays any matches. 
+
+`xargs' builds and executes command lines by gathering together arguments it reads on the standard input. Most often, these arguments are lists of file names generated by `find'
+</description>
+  <icon href="https://raw.githubusercontent.com/0install/0install.de-feeds/master/Gow.ico" type="image/vnd.microsoft.icon"/>
+  <icon href="https://raw.githubusercontent.com/0install/0install.de-feeds/master/Gow.png" type="image/png"/>
```
Comment (bastianeicher) [inline utils/findutils.xml:13]: Please update the icon URLs.

File: utils/findutils.xml (hunk 1 above)
Comment (pmiess) [reply utils/findutils.xml]: removed

File: utils/findutils.xml (hunk 2 above)
Comment (pmiess) [reply utils/findutils.xml]: moved

File: utils/findutils.xml (hunk 3 above)
Comment (pmiess) [reply utils/findutils.xml]: updated

**Pull request merged**